batch/batch/worker/worker.py

```diff
@@ -292,6 +293,14 @@ async def enable_iptables_forwarding(self):
 iptables -w {IPTABLES_WAIT_TIMEOUT_SECS} --append FORWARD --out-interface {self.veth_host} --in-interface {self.veth_host} --jump ACCEPT'''
         )
 
+    async def mark_packets(self):
+        await check_shell(
+            f'''
+iptables -t mangle -A PREROUTING --in-interface {self.veth_host} -j MARK --set-mark 10
+iptables -t mangle -A POSTROUTING --out-interface {self.veth_host} -j MARK --set-mark 11
```

Comment:
Do you know what happens to this table after the veths are deleted? I would hope that those rows get removed as part of a sort of cascading delete but would be good to verify we don't need to do any sort of cleanup here.